### PR TITLE
Documentation - Improve installation instructions

### PR DESCRIPTION
**Pull request by notchloeno**
Link to a forum post has rotted, so I've replaced it with a web archive link to the original site.

**When merged this pull request will:**
- Describe what this pull request will do
- Each change in a separate line
- Respect the [SQF](http://acre2.idi-systems.com/wiki/development/coding-guidelines-sqf) and [C++](http://acre2.idi-systems.com/wiki/development/coding-guidelines-cpp) Coding Guidelines
- Follow title standard `System - Add|Fix|Improve|Change|Make|Remove radios`

### CODE REVIEW

**Comment by jonpas** [comment]
I would rather see that link removed and possibly move the Manual section below Steam, it is far less used today.

**Comment by notchloeno** [comment]
> I would rather see that link removed and possibly move the Manual section below Steam, it is far less used today.

Donezo

File: docs/wiki/user/installation.md [hunk 1]
```diff
@@ -2,24 +2,25 @@
...
+
+
 ### Manual
 
 - Download from the [latest version](https://github.com/IDI-Systems/acre2/releases/latest) and extract the `.zip` file to your hard drive.
 - Navigate to your Arma 3 installation folder.
 - Delete any current `@acre2` folder and then paste in the `@acre2` folder you from the extracted `.zip`.
-- Create a new shortcut or edit an existing one, and add `@acre2` and `@CBA_A3` to the `-mod` parameter (for a longer explanation see [this tutorial](https://web.archive.org/web/20210218115054/https://www.armaholic.com/forums.php?m=posts&q=20866))
+- Create a new shortcut for Arma 3, or edit an existing one, and add `@acre2` and `@CBA_A3` to the `-mod` parameter
 - Launch Arma 3 from the shortcut you created.
```
Comment (jonpas) [inline docs/wiki/user/installation.md]:
```suggestion
- (option 1) Launch Arma 3 Launcher and add a local mod.
- (option 2) Create a new shortcut for Arma 3, or edit an existing one, and add `@acre2` and `@CBA_A3` to the `-mod` parameter
- Launch Arma 3 through Arma 3 Launcher (option 1) or the shortcut you created (option 2).
```

**Comment by jonpas** [comment]
Should probably mention Arma 3 Launcher exists as well while at it. Otherwise looks good, thanks!

**Comment by notchloeno** [comment]
Added some more detailed instructions and split into two separate bullet lists for clarity :)

**Review by PabstMirror — APPROVED** [review]
lgtm

**Pull request merged**